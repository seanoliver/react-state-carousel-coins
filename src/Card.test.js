import { render, fireEvent } from '@testing-library/react';
import Card from './Card';
import TEST_IMAGES from './_testCommon.js';

// smoke test
it('renders the Carousel component', function () {
	render(
		<Card
			caption="test image"
			src="img.png"
			currNum="1"
			totalNum="3"
		/>
	);
});

// snapshot test
it('matches the initial snapshot', function () {
	const { container } = render(
		<Card
			caption="test image"
			src="img.png"
			currNum="1"
			totalNum="3"
		/>
	);
	expect(container).toMatchSnapshot();
});
