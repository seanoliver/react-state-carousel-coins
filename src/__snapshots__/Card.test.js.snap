// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the initial snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test image
    </h4>
    <img
      alt="test image"
      class="Card-image"
      src="img.png"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      3
      .
    </small>
  </div>
</div>
`;
